skimage/feature/tests/test_util.py

```diff
@@ -117,6 +117,17 @@ def test_plot_matched_features(shapes):
         matches=matches,
         matches_color='r',
     )
+    plot_matched_features(
+        img0,
+        img1,
+        ax=ax,
+        keypoints0=keypoints0,
+        keypoints1=keypoints1,
+        matches=matches,
+        matches_color=[
+            np.random.choice(['C0', '#abc', 'aquamarine']) for _ in range(len(matches))
+        ],
+    )
```

Comment:
The tests should probably also cover passing a single matplotlib color if we support this?

Reply:
There is already a unit test which tests the single color:
https://github.com/scikit-image/scikit-image/blob/107c1a9a2c49f86668c41e8e5a1c101d71ff122f/skimage/feature/tests/test_util.py#L118

Reply:
Ah, my bad. When writing this I hadn't really groked that `matches_color` is already supported. Nevermind then. :)